bundler/lib/bundler/cli/install.rb

```diff
@@ -19,6 +19,10 @@ def run
       # Disable color in deployment mode
       Bundler.ui.shell = Thor::Shell::Basic.new if options[:deployment]
 
+      if target_rbconfig_path = options[:"target-rbconfig"]
+        Bundler.rubygems.set_target_rbconfig(target_rbconfig_path)
```

Comment:
This should probably fail more gracefully when the underlying RubyGems method is not available?